python/pysmurf/client/util/smurf_util.py

```diff
@@ -1393,7 +1385,7 @@ def make_mask_lookup(self, mask_file, mask_channel_offset=0,
 
         for gcp_chan, smurf_chan in enumerate(mask):
             b = int(smurf_chan//512)
-            ch = int((smurf_chan-mask_channel_offset)%512)
+            ch = int((smurf_chan)%512)
```

Comment:
Should we remove the hardcoded 512?

Reply:
Removed 512 hardcodes in smurf_util submodule - https://github.com/slaclab/pysmurf/pull/454/commits/62df701d2201ca7466a8a8e774956ea5839d831c.